website/docs/getting-started/getting-started-guide.md

```diff
@@ -54,21 +53,22 @@ ksctl cred
 
 ```
 
-Then, authenticate it with the respected cloud provider. In this case, we are using Civo, so we will type 3 
-paste your API key when prompted into the command line and hit enter.
+> Note: This command is used to configure the credentials for your slected cloud provider. It will prompt you to enter the authentication token or credentials specific to your provider.
 
-You should then see the following success message:
+Further copy the token or enter the required credentials when prompted by the command to complete the authentication process.
+
+After successful authentication, you should see a confirmation message similar to the following:
 
 :::success success
 [INFO] 💾 configuration
 
 Login Success
 :::
-Now, ksctl is connected to your civo account. We can now move ahead and create a cluster.
+Now, ksctl is connected to your cloud provider account. You can proceed to create a cluster.
 
 ## Create a Cluster
 
-Ensure that you are connected the correct cloud provider. After adding your cloud cred to ksctl, 
+Ensure that you are connected the correct cloud provider. After adding your cloud credentials to ksctl, 
 you can use the following CMD to create a cluster.
 
 ```bash
```

Comment:
I think the below 
bash command block not required (as it will change any new providers added we need to update it as well